models/fixtures/email_address.yml

```diff
@@ -276,4 +276,12 @@
   email: user2-2@example.com
   lower_email: user2-2@example.com
   is_activated: false
-  is_primary: false
\ No newline at end of file
+  is_primary: false
+
+-
+  id: 36
+  uid: 36
+  email: 1290147055@qq.com
+  lower_email: 1290147055@qq.com
```

Comment:
Please do not use personal information

Reply:
I don't see this too strict, but if it's easily fixed, do it.

Reply:
Done